### PR TITLE
AC Automated Test - LRAC-8375 | master

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://issues.liferay.com/browse/LRAC-8375

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsBlogs.testcase [hunk 1]
```diff
@@ -119,12 +119,83 @@ definition {
...
+			Pause(locator1 = "5000");
+		}
+
+		ACUtils.closeAllSessions();
+
+		Pause(locator1 = "30000");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsBlogs.testcase:160]: hi @catarineaguiar , next time, please try to use tasks to describe the actions you are performing, like "launch AC and switch to the side property" ...

"Go to segments and create a static segment with x user"

etc.

Thanks!

cc: @ViniLopes20

**Review by alexcandidos — COMMENTED** [review]
Hi @catarineaguiar , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**